packages/repository/src/__tests__/unit/decorator/relation.decorator.unit.ts

```diff
@@ -198,6 +198,55 @@ describe('relation decorator', () => {
         },
       });
     });
+
+    it('accepts additional property metadata', () => {
+      @model()
+      class AddressBook extends Entity {
+        @property({
+          id: true,
+          length: 36,
+          postgresql: {
+            dataType: 'uuid',
+          },
+        })
+        id: string;
+      }
+      @model()
```

Comment:
Please add an empty line to visually delimit the two model classes.

```
}

@model()
```

Reply:
done